src/api/tests/products/smoke&signInClient.test.ts

```diff
@@ -0,0 +1,66 @@
+import _ from 'lodash';
+import { generateNewProduct } from '../../../data/products/generateProduct.js';
+import { IProduct } from '../../../data/types/product.types.js';
+import { adminCredentials } from '../../../data/credentials/adminCreds.js';
+import { SignInApiClient } from '../../clients/signInApiClient.js';
+// import { ICredentials } from '../../../data/credentials/validLoginCreds.js';
+
+describe('[API] [Products] Smoke', () => {
+  const baseUrl = 'https://aqa-course-project.app/';
+  const endpoints = {
+    login: '/api/login/',
+    product: '/api/products/',
+  };
+  let token = '';
+  const credentials = {
+    username: adminCredentials.username,
+    password: adminCredentials.password,
+  };
+
+  let createdProduct: {
+    Product: IProduct & { _id: string; createdOn: string };
```

Comment:
IProductFromResponse
```suggestion
    Product: IProductFromResponse
```